src/htmlminifier.js

```diff
@@ -176,14 +173,7 @@
 
   // https://mathiasbynens.be/demo/javascript-mime-type
   // https://developer.mozilla.org/en/docs/Web/HTML/Element/script#attr-type
-  var executableScriptsMimetypes = createMap([
-    'text/javascript',
-    'text/ecmascript',
-    'text/jscript',
-    'application/javascript',
-    'application/x-javascript',
-    'application/ecmascript'
-  ]);
+  var executableScriptsMimetypes = createMap('text/javascript,text/ecmascript,text/jscript,application/javascript,application/x-javascript,application/ecmascript');
```

Comment:
Hmmm... not very readable is my only concern. Everything else looks great. Maybe add support for passing either string or an array to createMap?